dll/win32/kernel32/winnls/nls/ptg.nls

```diff
@@ -101,12 +101,12 @@ STRINGTABLE LANGUAGE LANG_PORTUGUESE, SUBLANG_PORTUGUESE
   LOCALE_SDECIMAL ","
   LOCALE_SENGCOUNTRY "Portugal"
   LOCALE_SENGCURRNAME "Euro"
-  LOCALE_SENGLANGUAGE "Portuguese"
+  LOCALE_SENGLANGUAGE "Português"
```

Comment:
This is not localizable.

https://learn.microsoft.com/en-us/windows/win32/intl/locale-seng-constants

> `LOCALE_SENGLANGUAGE` | Full English name of the language from ISO Standard 639.